src/utils/textUtils.js

```diff
@@ -1,4 +1,4 @@
-import path from 'path'
+import path from 'node:path'
```

Comment:
Which node Version introduced this? Don't wanna loose to much compatibility

Reply:
Here's the info on protocol specifiers: https://2ality.com/2021/12/node-protocol-imports.html